tests/xml_to_json_test.py

```diff
@@ -0,0 +1,47 @@
+import json
+import os
+from pathlib import Path
+
+import xmltodict
+
+from utils.change_dict import change_value_by_key_world, get_values_by_key
+
+
+class TestXMLToJson:
+
+    def test_xml_to_json(self):
```

Comment:
You did a great job, this "end to end test" looks good. 
For better troubleshooting and quicker issue identification, I would like to suggest to add smaller tests. 
(Don't delete this test)